Lib/test/test_dict.py

```diff
@@ -1071,6 +1071,62 @@ def test_valuesiterator_pickling(self):
             values = list(it) + [drop]
             self.assertEqual(sorted(values), sorted(list(data.values())))
 
+    def test_reverseiterator_pickling(self):
+        for proto in range(pickle.HIGHEST_PROTOCOL + 1):
+            data = {1:"a", 2:"b", 3:"c"}
+            it = reversed(data)
+            d = pickle.dumps(it, proto)
+            it = pickle.loads(d)
+            self.assertEqual(sorted(it), sorted(data))
```

Comment:
Why `sorted`?  Can't we use `list`?

Reply:
I look at the other tests like `test_valuesiterator_pickling` to write my own, they were certainly written before dict were ordered.

It would indeed be probably better to test with `list`, after all, we do respect order now.
I will change the tests I've written. Do you want me to change the other tests as well or prefer not to?

Reply:
If you can.  Not mandatory.